Tools/TestWebKitAPI/Tests/IPC/IPCTestUtilities.h

```diff
@@ -119,8 +119,17 @@ class MockConnectionClient final : public IPC::Connection::Client {
         m_continueWaitForMessage = true;
     }
 
-    bool didReceiveSyncMessage(IPC::Connection&, IPC::Decoder&, UniqueRef<IPC::Encoder>&) override
+    // Handler contract as IPC::MessageReceiver::didReceiveSyncMessage: false on invalid message, may adopt encoder,
+    // decoder used only during the call, if encoder not adopted it will be submitted.
```

Comment:
What does "submitted" mean here?

Reply:
Sending the encoder, e.g. sending a message